p2p/types/peers_view_test.go

```diff
@@ -105,6 +103,62 @@ func TestSortedPeersView_Remove(t *testing.T) {
 	t.Skip("TECHDEBT(#554): test that this method works as expected when target peer/addr is not in the list!")
 }
 
+func TestSortedPeersView(t *testing.T) {
+	testCases := []struct {
+		name              string
+		selfAddr          string
+		initialAddrs      string
+		expectedSortOrder string
+	}{
+		{
+			name:              "lowest self address",
+			selfAddr:          "A",
+			initialAddrs:      "BDCEA",
+			expectedSortOrder: "ABCDE",
+		},
+		{
+			name:              "highest self address",
+			selfAddr:          "E",
+			initialAddrs:      "BDCEA",
+			expectedSortOrder: "EABCD",
+		},
+		{
+			name:              "middle self address",
+			selfAddr:          "C",
+			initialAddrs:      "BDCEA",
+			expectedSortOrder: "CDEAB",
+		},
+	}
+	for _, testCase := range testCases {
+		t.Run(testCase.name, func(t *testing.T) {
+			selfAddr := cryptoPocket.Address(testCase.selfAddr)
+
+			pstore := make(PeerAddrMap)
+			var initialPeers []Peer
```

Comment:
Confirming that this works?

<img width="702" alt="Screenshot 2023-04-14 at 1 48 19 PM" src="https://user-images.githubusercontent.com/1892194/232151797-3b55e473-4427-4e2f-bfe3-cf5640f0b020.png">

Reply:
I don't recall thinking about this when writing it but it's a good distinction to make and one that seems easy to miss. In this case it doesn't matter because despite being nil initially, it is still a valid argument to the `append()` built-in, which does what you would expect.